tests/lib/ocpp/v201/test_charge_point.cpp

```diff
@@ -261,6 +288,68 @@ class ChargePointFixture : public DatabaseTestingUtils {
  * is provided.
  */
 
+TEST_F(ChargePointFixture, CreateChargePoint) {
+    auto evse_connector_structure = create_evse_connector_structure();
+    auto database_handler = create_database_handler();
+    auto evse_security = std::make_shared<EvseSecurityMock>();
+    configure_callbacks_with_mocks();
+    auto message_queue = create_message_queue(database_handler);
+
+    EXPECT_NO_THROW(ocpp::v201::ChargePoint(evse_connector_structure, device_model, database_handler, message_queue,
+                                            "/tmp", evse_security, callbacks));
+}
+
+TEST_F(ChargePointFixture, CreateChargePoint_EVSEConnectorStructureDefinedBadly_ThrowsDeviceModelStorageError) {
```

Comment:
Reminder to add a comment here since the test names don't reference particular functional groups (aka "this is testing shared functionality .... ")

Reply:
This is a test for business logic that's required for the smart charging handler to work as expected but that does not correspond to any functional requirement(s) from OCPP 2.0.1. Our convention for such a test is to describe the behaviors being tested in its name.

Are you suggesting that we also add the same comment to every unit test that isn't labeled as validating a particular functional requirement from the standard?

Reply:
In my mind, unit tests are validating code, sometimes code that is supportive infrastructure for the rest of the code, and as @drmrd said, when it happens to be also validating functional requirements, we already specify them in the unit test name. I think convention says that if the functional requirement is not specified that it can be presumed that the code being tested isn't necessarily tied to any one functional requirement. I don't see the value of a comment stating it?

Reply:
I understand that this is not related to a particular FR, which is why my suggested comment was "this is testing shared functionality...." I was hoping for something like 

```
/* BEGIN shared functionality tests */
TEST_F(... CreateChargePoint_EVSEConnectorStructureDefinedBadly_ThrowsDeviceModelStorageError)
TEST_F(....CreateChargePoint_...)
TEST_F(...CreateChargePoint_...)
/* END shared functionality tests */
```

However, if you can link to this convention

> I think convention says that if the functional requirement is not specified that it can be presumed that the code being tested isn't necessarily tied to any one functional requirement.

I am happy to waive that feedback.

Reply:
For the convention we updated the issue that was created [here](https://github.com/EVerest/libocpp/issues/467) under Other Tests. This is following the ideas of Clean Code:

> Don't be afraid to make a name long. A long descriptive name is better than a short enigmatic name. A long descriptive name is better than a long descriptive comment.
— Martin C. Fowler, Clean Code